README.md

```diff
@@ -16,6 +16,7 @@ I've also editted the raw create react app output so that;
 - We can use SASS
 - We have some CircleCi configuration for automatically running our tests.
 - We have this README!
+- You need to add a file called ".env" to the root directory, and populate REACT_APP_NASA_API_KEY with your API key (i.e. REACT_APP_NASA_API_KEY=xxxxxxxxxx). Add the .env file to your .gitignore (so that it doesn't get committed).
```

Comment:
Ah sorry can you actually commit your updated .gitignore file to this PR and then remove the part about adding .env to your .gitignore please? My bad for not being clear there but that can be committed :)